httpclient/dnscache/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
~ these stand alone doc.go's with one line are not really needed - the package comment can just be in the main file of the package - e.g. in httpclient/dnscache/dnscache.go

doc.go - would normally be used if the package comment is much bigger